outputs/s3/output.facets.yaml

```
@@ -0,0 +1,26 @@
+name: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)